#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_report_bugs_as_debian.dpatch by Luciano Bello <luciano@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Indicates that the version cames from a Debian Package 
## DP: TODO: include the Debian Package Version

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' trunk~/core/controllers/misc/get_w3af_version.py trunk/core/controllers/misc/get_w3af_version.py
--- trunk~/core/controllers/misc/get_w3af_version.py	2011-05-25 09:44:58.000000000 -0300
+++ trunk/core/controllers/misc/get_w3af_version.py	2011-05-30 11:33:29.000000000 -0300
@@ -47,6 +47,8 @@
         res += ' (from SVN server)'
         res += '\nRevision: ' + str(revision)
     else:
-        res += ' (from tgz)'
+        res += ' (from Debian Package '
+        import commands
+        res += commands.getstatusoutput('dpkg -p w3af-console | grep "Version:"')[1][9:] + ')'
     res += '\nAuthor: Andres Riancho and the w3af team.'
     return res
